src/configure-feature.js

```diff
@@ -1,19 +1,17 @@
 import React from 'react';
+import { curry } from 'ramda';
 import PropTypes from 'prop-types';
 import { isActive } from './is-active';
 
-const configureFeature = Default => featureName => (
-  Feature,
-  Fallback = Default
-) => {
+export const configureFeature = (Default, feature, Feature) => {
   const ConfigureFeatureHOC = (props, context) => {
```

Comment:
From the Readme

> `configureFeature` is a higher order component that allows you to configure a `Feature` component. configureFeature is auto curried so that you can partially apply the props.

We don't need to handle the logic here if we import the `Feature` component and just pass the arguments to the `Feature` component as props. It also means we could dump all the context references.